### ISSUE TITLE
Frequency Dictionaries

### ISSUE THREAD

**Issue by wJosuke** [birchill/10ten-ja-reader#972]
Hi Birchill,

This is my preferred pop-up dictionary, even on desktop. Unfortunately one of the things I like to do is check the frequency of words in order to decide whether or not it's worth creating a flash card on. It doesn't seem like a feature to use custom dictionaries is available, so are there any plans to incorporate frequency lists in the dictionary results (or perhaps a separate tab)? These results would indicate a ranking for a particular word in a particular use case (i.e Novels, Youtube, Anime, News) depending on the dictionary. 

Keep up the great work!

**Comment by birtles** [comment]
Hi Josuke,

Thanks very much for your kind comments.

We can certainly look at adding additional frequency lists. Do you have any pointers to particular frequency lists you'd like to see included?

Thanks again!

**Comment by wJosuke** [comment]
There are a few frequency lists used within the communities that I have seen but [this document](https://drive.google.com/drive/folders/1g1drkFzokc8KNpsPHoRmDJ4OtMTWFuXi) has all the ones that I have been using that cover specific types of content. I'd particularly like to see the Anime & J Drama list, Netflix, Youtube, BCCWJ LUW and the Novels list. I know some others might like to see the visual novel one as well. I'm not certain why there are two .zip files for many dictionaries. 


![image](https://user-images.githubusercontent.com/65164569/179491516-1edb78f1-744e-4a58-9715-ae14d51e3e40.png)
This is how the freq dictionaries appear on yomichan, others elect to put the number beside the word. 
and here is a document explaining each dictionary file
[Frequency - Dictionary Info.txt](https://github.com/birchill/10ten-ja-reader/files/9131128/Frequency.-.Dictionary.Info.txt)

Apologies for any inconvenience, and thanks again!

**Comment by birtles** [comment]
Thanks very much for those pointers. Unfortunately the `Frequency - Dictionary Info.txt` link didn't work for me.

That seems very promising indeed. I hope we can incorporate this.

**Comment by wJosuke** [comment]
Try [this one](https://drive.google.com/file/d/1f5UlOsMcHMWCCGFHTw7xAv1JYgxYo_HU/view?usp=sharing). 

I felt it important to mention that these files are found via [TheMoeWay resource list](https://learnjapanese.moe/resources/). I am unsure if you require permission or not to use the files for something like, but they have been available to the public to use freely with Yomichan. Best of luck to you!

**Comment by birtles** [comment]
> Try [this one](https://drive.google.com/file/d/1f5UlOsMcHMWCCGFHTw7xAv1JYgxYo_HU/view?usp=sharing).

Great, thank you!

> I felt it important to mention that these files are found via [TheMoeWay resource list](https://learnjapanese.moe/resources/). I am unsure if you require permission or not to use the files for something like, but they have been available to the public to use freely with Yomichan. Best of luck to you!

Oh, I see. Unfortunately TheMoeWay is known for linking to illegal resources. For example, the dictionary files linked to there are most certainly illegal. I wonder what the original source is for these files.

**Comment by wJosuke** [comment]
I will see if I can't find some official resources, then. My apologies. I do believe that many of the frequency lists based on novels, visual novels and such are community created but I understand why you might not feel comfortable using them.

**Comment by birtles** [comment]
Not at all! Thank you so much for all your help with this. Yes, if there is some official source that would be great.

**Comment by wJosuke** [comment]
Apologies for the rather late response. I managed to find an official source for a relatively popular frequency dictionary. 

https://clrd.ninjal.ac.jp/bccwj/en/freq-list.html

**Comment by birtles** [comment]
Thank you! I had a look at that list, but for commercial users apparently we'd need to pay 800万円 given how many users we have 😬
